src/modules/mahjong/Hand/test/PlayerHand.test.ts

```diff
@@ -93,3 +94,38 @@ test('PlayerHand - draw()', () => {
   expect(fullHand.getTiles()).toHaveLength(13);
   expect(fullHand.getTiles()).toStrictEqual(tiles);
 });
+
+test('PlayerHand - getFlowerNumber()', () => {
+  expect(emptyHand.getFlowerNumber()).toBe(1);
+  expect(fullHand.getFlowerNumber()).toBe(1);
+});
+
+test('PlayerHand - setFlowerNumber()', () => {
```

Comment:
kind of triggered because was taught to only have 1 expect for each test case

Reply:
Really? During your coop?

Reply:
Interesting :o @jzc1828 what did you do?

I never did much testing so this is what I've been doing without any guidance

Reply:
I usually do have a few `expect` to check the returned value of the function being tested and if the function calls some other function(s) within it, I will have `expect` for the function calls:
```ts
expect(funcSpy).toHaveBeenCalledTimes(numofTimes);
expect(funcSpy).toHaveBeenCalledWith(arguments);
```

Reply:
Yea was taught to have 1 expect per test case to ensure modularity. If you have multiple expects in a test case, you wouldn't know which expect statement fails during the test.